monkey/common/agent_configuration/default_agent_configuration.py

```diff
@@ -36,12 +38,12 @@
     9600,
 )
 
-TCP_SCAN_CONFIGURATION = TCPScanConfiguration(timeout=3.0, ports=TCP_PORTS)
+TCP_SCAN_CONFIGURATION = TCPScanConfiguration(timeout=3.0, ports=tuple(map(NetworkPort, TCP_PORTS)))
```

Comment:
Should we prefer using `typing.Tuple` or just `tuple`?

Reply:
I will make it a `Tuple[NetworkPort, ...]`

Reply:
[`typing.Tuple` is deprecated](https://docs.python.org/3/library/typing.html#typing.Tuple).